File: Simple string characters.py
import re
def solve(s):
    a = len(re.findall('[A-Z]',s))
    b = len(re.findall('[a-z]',s))
    c = len(re.findall('[0-9]',s))
    d = len(re.findall('[^0-9a-zA-Z]',s))
    return [a,b,c,d]
    

File: Simple string characters.js
function solve(s){
  let a = (s.match(/[A-Z]/g) || []).length
  let b = (s.match(/[a-z]/g) || []).length
  let c = (s.match(/[0-9]/g) || []).length
  let d = (s.match(/[^a-zA-Z0-9]/g) || []).length
  return [a,b,c,d]
}
